src/main/java/org/xstefank/whitelist/RetestCommand.java

```diff
@@ -0,0 +1,35 @@
+package org.xstefank.whitelist;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import java.util.List;
+import org.xstefank.ci.ContinuousIntegration;
+import org.xstefank.model.PersistentList;
+
+public class RetestCommand implements Command {
+
+    public static final String NAME = "RetestCommand";
+
+    private String commandRegex;
+
+    @Override
+    public void process(JsonNode payload, PersistentList adminList,
+                        PersistentList userList, List<ContinuousIntegration> CIs) {
+        String authorUsername = WhitelistProcessing.getAuthorUsername(payload);
+
+        if (WhitelistProcessing.isCommentAuthorAdministrator(payload, adminList) &&
```

Comment:
retest should be available to all enlisted users (in the user list), no need for the comment author to be admin